tests/regression/test_slate_hybridization.py

```diff
@@ -0,0 +1,141 @@
+"""Solve a mixed Helmholtz problem
+
+sigma + grad(u) = 0,
+u + div(sigma) = f,
+
+using hybridisation with SLATE performing the forward elimination and
+backwards reconstructions. The corresponding finite element variational
+problem is:
+
+dot(sigma, tau)*dx - u*div(tau)*dx + lambdar*dot(tau, n)*dS = 0
+div(sigma)*v*dx + u*v*dx = f*v*dx
+gammar*dot(sigma, n)*dS = 0
+
+for all tau, v, and gammar.
+
+This is solved using broken Raviart-Thomas elements of degree k for
+(sigma, tau), discontinuous Galerkin elements of degree k - 1
+for (u, v), and HDiv-Trace elements of degree k - 1 for (lambdar, gammar).
+
+The forcing function is chosen as:
+
+(1+8*pi*pi)*sin(x[0]*pi*2)*sin(x[1]*pi*2),
+
+which reproduces the known analytical solution:
+
+sin(x[0]*pi*2)*sin(x[1]*pi*2)
+"""
+
+
+import pytest
+from firedrake import *
+
+
+@pytest.mark.parametrize("degree", range(1, 3))
+def test_slate_hybridization(degree):
+    # Create a mesh
+    mesh = UnitSquareMesh(8, 8)
+
+    # Create mesh normal
+    n = FacetNormal(mesh)
+
+    # Define relevant function spaces
+    RT = FiniteElement("RT", triangle, degree)
+    BRT = FunctionSpace(mesh, BrokenElement(RT))
+    DG = FunctionSpace(mesh, "DG", degree - 1)
+    T = FunctionSpace(mesh, "HDiv Trace", degree - 1)
+
+    W = BRT * DG
+
+    # Define the trial and test functions
+    sigma, u = TrialFunctions(W)
+    tau, v = TestFunctions(W)
+    gammar = TestFunction(T)
+
+    # Define the source function
+    f = Function(DG)
+    x, y = SpatialCoordinate(mesh)
+    f.interpolate((1+8*pi*pi)*sin(x*pi*2)*sin(y*pi*2))
+
+    # Define finite element variational forms
+    Mass_v = dot(sigma, tau) * dx
+    Mass_p = u * v * dx
+    Div = div(sigma) * v * dx
+    Div_adj = div(tau) * u * dx
+    local_trace = gammar('+') * dot(sigma, n) * dS
+    L = f * v * dx
```

Comment:
To assuage my doubts about mixed coefficients, can you define: `one = Function(W); one.interpolate(1)` and then multiply all the terms by an appropriately split `one`?